src/functionaltests/resources/SubscriptionObject.json

```diff
@@ -4,7 +4,7 @@
         "repeat": false,
         "created": "data-time",
         "notificationType": "REST_POST",
-        "notificationMeta": "http://localhost:{port}/missed_notification",
+        "notificationMeta": "http://localhost:{port}/notification-endpoint",
```

Comment:
Ah here we have the endpoint, this one is supposed to fail, so maybe name it "invalid_endpoint"

Reply:
Or rather do not use subscriptionObject.json and use Commons.

Reply:
Changed it to a less confusing name.